javascript/chronometer.js

```diff
@@ -1,34 +1,58 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime=0,
+    this.intervalId= null
   }
 
-  start(callback) {
-    // ... your code goes here
-  }
+  start(callback) 
```

Comment:
falto hacer falta la ejecución del callback 
```suggestion
start(callback){
 this.intervalId = setInterval( () => {
      this.currentTime++;
      if (printTime) printTime();
    }, 1000);
}
```